redhat-appstudio/service-provider-integration-operator
# Make the TokenUploadReconciler behave more in-line with other reconcilers

### What does this PR do?
Make the TokenUploadReconciler behave more in-line with other reconcilers in our codebase.

Most importantly - reconcile the secrets one at a time, which also enables slightly more clear error handling.

### What issues does this PR fix or reference?
none

### How to test this PR?
No change in behavior should be apparent.

```
quay.io/redhat-appstudio/pull-request-builds:spi-controller-#
quay.io/redhat-appstudio/pull-request-builds:spi-oauth-#
```
